atd-vze/src/queries/crashes.js

```diff
@@ -118,6 +118,21 @@ export const GET_CRASH = gql`
   }
 `;
 
+export const GET_CRASH = gql`
+  query CrashDetails($crashId: Int!) {
+    crashes_by_pk(crash_id: $crashId) {
+      crash_id
+      address_primary
+      address_secondary
+      crash_injury_metrics {
+        cris_fatality_count
```

Comment:
instead of `cris_fatality_count`, we should use `vz_fatality_count`. my bad—i typo'd this in the issue description